crates/ruff_linter/src/rules/flynt/helpers.rs

```diff
@@ -15,7 +15,7 @@ fn to_f_string_expression_element(inner: &Expr) -> ast::FStringElement {
 /// Convert a string to a [`ast::FStringElement::Literal`].
 pub(super) fn to_f_string_literal_element(s: &str) -> ast::FStringElement {
     ast::FStringElement::Literal(ast::FStringLiteralElement {
-        value: s.to_string().into_boxed_str(),
+        value: s.into(),
```

Comment:
nit: explicit `from` calls are often more readable than `.into()` calls, as you can easily see what type the object is being converted into

```suggestion
        value: Box::from(s),
```